www/core/test/extension_test.xml

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" ?>
 <updates>
+	<!--
```

Comment:
commented out?

Reply:
we don't have a 4.4.1-rc or 5.0.1-rc yet but if this is here it would suggest to download the non existing file, at least that's my expectation

Reply:
Yes please revert the test_extension und test_list update server for now as that is something that I can prepare for 4.4.1-rc/5.0.1-rc release. With such an commeted out thing i expect the update server to fail the validation and show errors. But when we revert it the test update server will not provide a newer version and there for also no errors.